tests/indent/common.lua

```diff
@@ -140,6 +140,17 @@ function M.indent_new_line(file, spec, opts, xfail)
   table.insert(before, spec.on_line + 1, indent .. spec.text)
 
   compare_indent(before, after, xfail)
+
+  before, after = M.run_indent_test(file, function()
+    -- move to the line and input the new one
+    vim.cmd(string.format("normal! %dG$", spec.on_line))
+    vim.cmd(string.format(vim.api.nvim_replace_termcodes("normal! a<cr>%s", true, true, true), spec.text))
```

Comment:
also test starting a new line with `<cr>` in insert mode vs the already test `o` in normal mode